packages/azure-http-specs/specs/azure/resource-manager/resources/location.tsp

```diff
@@ -0,0 +1,174 @@
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    ```
+    """)
+  get is ArmResourceRead<LocationResource>;
+
+  @scenario
+  @scenarioDoc("""
+    Resource PUT operation.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+    Expected query parameter: api-version=2022-12-01-preview
+    Expected request body:
+    ```json
+    {
+      "properties": {
+        "description": "valid",
+      }
+    }
+    ```
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource",
+      "name": "resource",
+      "type": "Azure.ResourceManager.Resources/locationResources",
+      "properties": {
+        "description": "valid",
+        "provisioningState": "Succeeded",
+      },
+      "systemData": {
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    ```
+    """)
+  createOrUpdate is ArmResourceCreateOrReplaceSync<LocationResource>;
+
+  @scenario
+  @scenarioDoc("""
+    Resource PATCH operation.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected request body:
+    ```json
+    {
+      "properties": {
+        "description": "valid2",
+      }
+    }
+    ```
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource",
+      "name": "resource",
+      "type": "Azure.ResourceManager.Resources/locationResources",
+      "properties":{
+        "description": "valid2",
+        "provisioningState": "Succeeded"
+      },
+      "systemData": {
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    ```
+    """)
+  update is ArmResourcePatchSync<LocationResource, LocationResourceProperties>;
+
+  @scenario
+  @scenarioDoc("""
+    Resource DELETE operation.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected response status code: 204
+    """)
+  delete is ArmResourceDeleteSync<LocationResource>;
+
+  @scenario
+  @scenarioDoc("""
+    Resource LIST by resource group operation.
```

Comment:
Description is not correct?

Just "Resource List operation." or "Resource List operation by location.".

Reply:
Fixed by the [commit#a0864e9](https://github.com/Azure/typespec-azure/pull/1932/commits/a0864e9159b367380e354317f0dabebc97a579e9)